doc/src/devdocs/ast.md

```diff
@@ -53,8 +53,8 @@ call. Finally, chains of comparisons have their own special expression structure
 | `a&&b`      | `(&& a b)`                |
 | `x += 1`    | `(+= x 1)`                |
 | `a ? 1 : 2` | `(if a 1 2)`              |
-| `a:b`       | `(: a b)`                 |
-| `a:b:c`     | `(: a b c)`               |
+| `a:b`       | `(call : a b)`            |
+| `a:b:c`     | `(call : a b c)`          |
```

Comment:
This seems to now be already covered by the statement above the table that it is parsed as a call if it is not in the table.
```suggestion
```